src/localconf.js

```diff
@@ -101,11 +101,15 @@ function buildConfig(opts) {
   const workspaceLaunchUrl = `${workspaceUrl}launch`;
   const datasetUrl = `${hostname}api/search/datasets`;
   const guppyUrl = `${hostname}guppy`;
-  const guppyGraphQLUrl = `${guppyUrl}/graphql/`;
+  const guppyGraphQLUrl = `${guppyUrl}/graphql`;
   const guppyDownloadUrl = `${guppyUrl}/download`;
   const manifestServiceApiPath = typeof manifestServiceURL === 'undefined' ? `${hostname}manifests/` : ensureTrailingSlash(manifestServiceURL);
   const requestorPath = typeof requestorURL === 'undefined' ? `${hostname}requestor/` : ensureTrailingSlash(requestorURL);
   const auspiceUrl = `${protocol}//auspice.${hostnameOnly}/covid19`;
+  const workspaceStorageUrl = `${hostname}ws-storage`;
+  const workspaceStorageListUrl = `${workspaceStorageUrl}/list`;
+  const workspaceStorageDownloadUrl = `${workspaceStorageUrl}/download`;
+  const marinerUrl = `${hostname}/ga4gh/wes/v1/runs`;
```

Comment:
I believe this variable contains two slashes in a row -- it ends up looking like this: https://qa-mickey.planx-pla.net//ga4gh/wes/v1/runs

Reply:
oops good catch!